### ISSUE TITLE
Noun chunking missing root nouns?

### ISSUE THREAD

**Issue by jmugan** [spacy-io/spaCy#469]
When I run the noun chunker over the phrase "100 tacos with a side of rice" it returns "a side" and "rice" but not "100 tacos".

The word "tacos" has the dependency label (dep_) of "ROOT". I think the problem may be that `english_noun_chunks` in `spacy.syntax.iterators` has 'root' as lowercase. See line 5
```
labels = ['nsubj', 'dobj', 'nsubjpass', 'pcomp', 'pobj',
              'attr', 'root']
```

When I add 'ROOT' to `labels` it works as expected and returns "100 tacos". Of course, there may be a reason that 'root' is different from 'ROOT' that I am not aware of.

**Comment by honnibal** [comment]
Definitely a bug.

Would you mind submitting a pull request with a test and the patch?

**Comment by jmugan** [comment]
Will do!

**Comment by jmugan** [comment]
I put in the change and the test, but when I try to build locally with `python setup.py build_ext --inplace` I get the error listed at http://9.media.readthedocs.io/projects/spacy/builds/3893054/

**Comment by honnibal** [comment]
Broken build is sadness — I want to make sure this doesn't happen in future.

Anyway. The problem is fixed for 1.0. Thanks again for the report.
